### ISSUE TITLE
OpenSSL vulnerability with BN_mod_sqrt() function at High Severity

### ISSUE THREAD

**Issue by David-Krupa** [kubevirt/kubevirt#7403]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
Discovered high severity vulnerability in the openssl-libs package. See release notes here: https://www.openssl.org/news/openssl-1.1.1-notes.html

The vulnerability resides in the function BN_mod_sqrt(). This function, for computing square roots contains a bug that could cause it to loop indefinitely for non-prime moduli. This function is used internally when parsing a certificate that contains an elliptic curve public key in compressed form or an explicit elliptic curve parameter with a base point encoded in compressed form.

**What you expected to happen**:

Fix is an upgrade to  openssl-libs-1:1.1.1n in appropriate files by bumping the versions up.

**How to reproduce it (as minimally and precisely as possible)**:

Container image security scanner (Anchore) checked for posted vulnerabilities and would be able to detect the bug with the level of severity.

Can also be found here https://nvd.nist.gov/vuln/detail/CVE-2022-0778 in the national vulnerability database.

**Anything else we need to know?**:
No.

**Environment**:
- KubeVirt version (use `virtctl version`): v0.51.0
- Kubernetes version (use `kubectl version`): 1.21
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by rmohr** [comment]
Hi @David-Krupa how did you detect this issue? Did you run a security scanner?

**Comment by David-Krupa** [comment]
Yes, I also found it on the national vulnerabilities database and I googled it from openssl-lines CVE 0778. I think was the CVE number. Same thing with the other golang CVEs I put an issue in a week or two ago. I am trying to convince my company to allow us to help contribute but have to work through some bureaucracy to avoid any conflicts of interest. We really want to incorporate KubeVirt in the systems and these CVEs are the only things holding us up lol.

**Comment by David-Krupa** [comment]
Openssl-libs note lines. Autocorrect.

**Comment by rmohr** [comment]
There is still no fix on centos8-stream for that. Does not seem like it is treated with high priority: https://access.redhat.com/security/cve/cve-2022-0778

**Comment by rmohr** [comment]
Never mind. It is fixed in https://centos.pkgs.org/8-stream/centos-baseos-aarch64/openssl-libs-1.1.1k-6.el8.aarch64.rpm.html:


```
2022-03-23 - Clemens Lang <cllang@redhat.com> - 1:1.1.1k-6
- Fixes CVE-2022-0778 openssl: Infinite loop in BN_mod_sqrt() reachable when parsing certificates
- Resolves: rhbz#2067144
```

/close

Fixed in #7277 two weeks ago.
